interface/weno/wenoconnected.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ *  @package OpenEMR
+ *  @link    http://www.open-emr.org
+ *  @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @copyright Copyright (c) 2020 Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require_once("../globals.php");
+
+//ensure user has proper access
+if (!AclMain::aclCheckCore('admin', 'super')) {
+    echo xlt('ACL Administration Not Authorized');
+    exit;
+}
```

Comment:
the above code block needs to be below the `use` code block

Reply:
done